jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash

```diff
@@ -13,10 +13,10 @@ export HOMEBREW_MAKE_JOBS=${MAKE_JOBS}
 PROJECT=$1 # project will have the major version included (ex gazebo2)
 PROJECT_ARGS=${2}
 
-# In ignition projects, the name of the repo and the formula does not match
+# TODO(chapulina) Use gz path instead of legacy ign
```

Comment:
Any particular reason not to use the new gz- and use the legacy instead?

Reply:
That was my first approach, and the test run showed that more things would need to be changed, so I though of keeping this PR to the strictly necessary for now.